app/src/pages/inside/organizationProjectsPage/organizationProjectsPage.jsx

```diff
@@ -0,0 +1,48 @@
+/*!
+ * Copyright 2024 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { useSelector } from 'react-redux';
+import { userRolesSelector } from 'controllers/user';
+import { canUpdateSettings } from 'common/utils/permissions';
+import classNames from 'classnames/bind';
+import {
+  organizationProjectsSelector,
+  organizationsListLoadingSelector,
+} from 'controllers/organizations';
+import { SpinningPreloader } from 'components/preloaders/spinningPreloader';
+import { Header } from './header';
+import { EmptyProjectsState } from './emptyProjectsState';
+import styles from './organizationProjectsPage.scss';
+
+const cx = classNames.bind(styles);
+
+export const OrganizationProjectsPage = () => {
+  const userRoles = useSelector(userRolesSelector);
+  const hasPermission = canUpdateSettings(userRoles);
+  const orgProjects = useSelector(organizationProjectsSelector);
+  const organizationLoading = useSelector(organizationsListLoadingSelector);
+  const isProjectsEmpty = orgProjects.length === 0;
+
+  return (
+    <>
+      {organizationLoading && <SpinningPreloader />}
```

Comment:
Just having a spinner? Should it replace something?

Reply:
It should appear in case of organization Loading state.